### ISSUE TITLE
optimize plugin setup

### ISSUE THREAD

**Issue by edaniszewski** [vapor-ware/synse-openconfig-plugin#8]
opening for tracking

if there are multiple OC endpoints configured, I believe it currently goes through them serially for setup. need to see if theres a way to optimize and do stuff in parallel. the initial setup can take a little time for a single configuration, so if we have multiple this adds up.

not something that needs to get done, but a nice to have
